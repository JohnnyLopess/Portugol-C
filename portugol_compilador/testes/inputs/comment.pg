programa
{
    inicio
        real numero
        //     este é um número real
        caractere letra
        inteiro idade

        /*este bloco tem operações de leia
        e escreva*/
        escreva("Digite um número real: ")
        leia(numero)
        escreva("Digite uma letra: ")
        leia(letra)
        //teste
        escreva("Digite sua idade: ")
        leia(idade)

        // resultado
        escreva(numero)
        escreva(letra)
        escreva(idade)
}