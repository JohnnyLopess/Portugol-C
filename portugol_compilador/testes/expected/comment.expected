#include <stdio.h>

int main() {
    float numero;
    //este é um número real
    char letra;
    int idade;
    /*este bloco tem operações de leia
 e escreva*/
    printf("Digite um número real: ");
    scanf("%f", &numero);
    printf("Digite uma letra: ");
    scanf(" %c", &letra);
    //teste
    printf("Digite sua idade: ");
    scanf("%d", &idade);
    //resultado
    printf("%f\n", numero);
    printf("%c\n", letra);
    printf("%d\n", idade);
    return 0;
}
